docker/Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /home/node
 # Temporary use local copy
 RUN mkdir fhir-works-on-aws-deployment
 COPY ./ ./fhir-works-on-aws-deployment/
+RUN rm -rf ./fhir-works-on-aws-deployment/.build
```

Comment:
Just to confirm if there isn't any .Build directories. This command does not error out￼ and fails the build

Reply:
Confirmed. `rm -rf` removes the directory if it's there, but doesn't error out if the directory isn't there.